src/serializableStateInvariantMiddleware.ts

```diff
@@ -59,6 +55,10 @@ export function findNonSerializableValue(
   for (const [property, nestedValue] of entries) {
     const nestedPath = path.concat(property)
 
+    if (ignoredPaths.indexOf(nestedPath.join('.')) >= 0) {
```

Comment:
As a tiny perf improvement, can we first check to see if `ignoredPaths` has a length > 0, outside the loop, and then check this as `if (hasIgnoredPaths && ignoredPaths.indexOf()` ?